scripts/project.js

```diff
@@ -57,11 +57,33 @@ $('#portfolio').on('click','.project-preview',function(){
 });
 
 
-$('header nav').on('click','li',function(){
+$('header nav').on('click','li',function(e){
+  e.preventDefault();
   var $scrollTo = $(this).data('link');
   $target = $('section[data-content="' + $scrollTo + '"]');
   console.log($target);
   $('html, body').animate({
-    scrollTop: $target.offset().top
+    scrollTop: $target.offset().top - 50
   }, 500);
 });
+
+var $headerPosition = $('header').offset().top;
+
+var stickyHeader = function() {
+  console.log ($headerPosition);
+  console.log('scroll detected');
+  console.log($(this).scrollTop());
```

Comment:
It's fine for staging, but remember to take these out before you push to master.